### PR TITLE
feat(protocol-designer): change copy for pipette missing tip error

### LINKED ISSUE #1815: PD: Change Tip should be selected by default, not blank
## current behavior

On creating a new Step form, "Change Tip" dropdown is blank

## steps to reproduce

Create any step (besides Pause)

## expected behavior

It should not be blank, but be set to the default value. This is currently set under `DEFAULT_CHANGE_TIP_OPTION` constant, but that value isn't going into the form

### LINKED ISSUE #1880: New copy for pipette missing tip error
As a user I would like the PD errors to be understandable to me

## Acceptance Criteria
- There is an existing error for when a step is set to "never" but there isn't a tip on the pipette at the start of the action. I've only seen this happen at the start of a protocol in the first step. 

## Copy
- Title: "No tip on pipette"
- Body: "For the first step in a protocol the "change tip" setting must be set to always or once.

### PR DESCRIPTION
**Pull request by IanLondon**
## overview

Closes #1880, Closes #1815

## changelog

-allow more control in overriding step-generations warnings/errors w/in PD context
-fix bug with "Change tip" being blank (#1815)

## review requests

- Change tip to "Never" in a step that is the first for that pipette - error copy should match #1880 
- In new steps, "Change Tip" should be set to "Always" (not blank)

### CODE REVIEW

File: protocol-designer/src/steplist/formProcessing.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const generateNewForm = (stepId: StepIdType, stepType: StepType): BlankFo
   if (stepType === 'transfer' || stepType === 'consolidate' || stepType === 'mix') {
     return {
       ...baseForm,
-      'aspirate_change-tip': 'once'
```
Comment (IanLondon) [inline protocol-designer/src/steplist/formProcessing.js]: this was one-line #1815 fix

File: protocol-designer/src/components/steplist/TimelineAlerts.js [hunk 2]
```diff
@@ -12,7 +12,17 @@ import {selectors as steplistSelectors} from '../../steplist'
...
+import type {
+  CommandCreatorError,
+  CommandCreatorWarning,
+  ErrorType,
+  WarningType
+} from '../../step-generation'
+
+type AlertContent = {
+  title: string,
+  body?: ?React.Node
```
Comment (mcous) [inline protocol-designer/src/components/steplist/TimelineAlerts.js]: Does `React.Node` need the maybe? I thought it already included `null`

File: protocol-designer/src/components/steplist/TimelineAlerts.js [hunk 3]
```diff
@@ -26,35 +36,68 @@ type DP = {
...
+  const errors = props.errors.map((error, key) => {
+    const {title, body} = getErrorContent(error)
+    return (
+      <AlertItem
+        type='warning'
+        key={`error:${key}`}
+        title={title}
+        onCloseClick={undefined}
+        >
+          {body}
```
Comment (mcous) [inline protocol-designer/src/components/steplist/TimelineAlerts.js]: If `body` is `undefined` will React complain?

File: protocol-designer/src/components/steplist/TimelineAlerts.js (hunk 2 above)
Comment (IanLondon) [reply protocol-designer/src/components/steplist/TimelineAlerts.js]: oh right I shouldn't have the maybe, good catch

File: protocol-designer/src/components/steplist/TimelineAlerts.js (hunk 3 above)
Comment (IanLondon) [reply protocol-designer/src/components/steplist/TimelineAlerts.js]: I've seen React complain about `undefined`s before (it does a console.log or warn or something) but I tried commenting out one of the `body` fields in `errorOverrides` and it didn't complain.

I think React doesn't like the `render` method of any component to return `undefined`, but you can do `const Spam = () => <div>{undefined}</div>` and it doesn't complain

**Pull request merged**